enclave/CMakeLists.txt

```diff
@@ -12,13 +12,18 @@ if (OE_SGX)
       ../common/sgx/report.c
       ../common/sgx/collateral.c
       ../common/sgx/sgxcertextensions.c
+      ../common/sgx/sgxmeasure.c
       ../common/sgx/tcbinfo.c
       ../common/sgx/tlsverifier.c
       ../common/sgx/verifier.c
       sgx/attester.c
       sgx/report.c
       sgx/collateralinfo.c
       sgx/start.S)
+  if (WITH_EEID)
+    list(APPEND PLATFORM_SRC ../common/sgx/eeid_verifier.c ../common/sgx/eeid.c
```

Comment:
Will only build for Linux

Reply:
I've added a check to the main CMakeLists.txt, so that it fails early on Windows.